.buildkite/build_resources.py

```diff
@@ -27,6 +36,9 @@
         for v in json.loads(content):
             if v['filename'].startswith('resources/'):
                 pipeline['steps'][0]['skip'] = False
+                pipeline['steps'][0]['build']['message'] = os.environ.get('BUILDKITE_MESSAGE')
```

Comment:
We should add a comment here about why is it not working to use the definition from the `pipeline` dictionary.

Can you also update the commit message to say "Fixed the build resources pipeline" in the summary, and then add details in the description as well with what was the problem (i.e. it was always running on main).